service/src/main/java/uk/gov/hmcts/reform/fpl/service/CaseDataExtractionService.java

```diff
@@ -235,4 +241,14 @@ String formatTitle(Direction direction, List<DirectionConfiguration> directions)
                 .formatLocalDateTimeBaseUsingFormat(direction.getDateToBeCompletedBy(),
                     dateFormattingConfig.getPattern()) : "unknown"));
     }
+
+    public List<Other> getOthers(CaseData caseData) {
```

Comment:
Wouldn't it be better if this method simply lived in CaseData? I remember someone added getAllOthers and then someone removed, but we clearly need this, what do you think?